spec/system/admin/reports_spec.rb

```diff
@@ -31,16 +31,7 @@
     end
   end
 
-  describe "Background processing", feature: :background_reports do
-    it "can run the customers report" do
-      login_as_admin
-      visit admin_report_path(report_type: :customers)
-      generate_report
-
-      expect(page).to have_selector "#report-table"
-      expect(page).to have_content "FIRST NAME LAST NAME BILLING ADDRESS EMAIL"
-    end
```

Comment:
This is tested below already. It was just duplicated here to test it with background reports as well which is now not necessary.